collection_size.go

```diff
@@ -22,6 +22,97 @@ type stats struct {
 	Size  int32 `bson:"size"`
 }
 
+// NewDBCollectionsSizeCollector returns a new collector that monitors sizes and object counts
+// of all collections in the specified database.
+func NewDBCollectionsSizeCollector(namespace string, db *mgo.Database) *dbCollectionsSizeCollector {
```

Comment:
I'm inclined to think that's better. We don't usually pass around mgo.Database so I'd imagine needing to create one just to pass to this function. But we don't actually use it, the first thing we do is create a new db using a copied session and the name from this one